src/pages/projectSettings/tabs/ContainersTab/transformers.ts

```diff
@@ -0,0 +1,35 @@
+import { ProjectSettingsTabRoutes } from "constants/routes";
+import { ProjectSettingsQuery, RepoSettingsQuery } from "gql/generated/types";
+import { FormToGqlFunction, GqlToFormFunction } from "../types";
+import { FormState } from "./types";
+
+type Tab = ProjectSettingsTabRoutes.Containers;
+
+export const gqlToForm: GqlToFormFunction<Tab> = (
```

Comment:
I learned recently that the `satisfies` operator can be great for typing functions more completely. I just used it in this PR https://github.com/evergreen-ci/spruce/pull/1802 if you want to check it out!